manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml

```diff
@@ -176,6 +176,7 @@ spec:
           - monitoring.coreos.com
           resources:
           - servicemonitors
+          - prometheusrules
```

Comment:
Fluentd PrometheusRules were merged to 4.2 (master-post-release branch).  Is it ok to put it in the master branch?

Reply:
good catch, i just assumed it needed to go in without seeing if it failed without it. i'll verify this works for master even without it